packages/notifi-react-card/lib/components/AlertHistory/AlertDetailsCard.tsx

```diff
@@ -36,7 +36,11 @@ export const AlertDetailsCard: React.FC<AlertDetailsProps> = ({
         </div>
       </div>
       <div className={clsx('NotifiAlertDetails__bottomContent')}>
-        <div>{detailsContents.bottomContent}</div>
+        <div>
+          {detailsContents.bottomContent.split('\n').map((i, key) => {
+            return <div key={key}>{i}</div>;
```

Comment:
nit: variable naming

```suggestion
          {detailsContents.bottomContent.split('\n').map((content, index) => {
            return <div key={index}>{content}</div>;
```

Reply:
fixed